### PR TITLE
General UI tweaks

### LINKED ISSUE #1528: [Request]: add a hotkey for clearing wanted level
### Problem

sometimes I want the cops sometimes I dont but I am lazy to open the menu and click the button twice

### Solution

a hotkey for clearing the wanted level

### Reason

a

### Additional context

_No response_

### LINKED ISSUE #1531: [Bug]: Field InputText Dictonaries at Debug not active
### Describe the bug

Field InputText search filter for Dictonaries in the Debug isnt active anymore after latest update. Can't enter any text, cant focus on that field. 

### Steps To Reproduce

1. Build and compile latest
2. Fully load then go to Debug then Animation tab
3. Fetch animations dicts
4. Try to input search filter
5. Field inactive

### Expected Behavior

Input text fields should work properly

### Screenshots and/or Logs

```
<paste your logs here>
```

### Store

Epic Games

### Additional context

InputText cursor is active for ~0.01s after tab is switched to Anims then immediately goes to inactive state

### Build

- [X] I've confirmed the issue exists on the latest version of YimMenu

### LINKED ISSUE #1534: [Bug]: Cannot set player wanted level
### Describe the bug

With the new UI, the effective space has become less than halved. The Slider takes up the full allocated width of the section. The button (according to the code) is on the same line. Given the only visible thing is the slider, you can no longer set wanted levels.

I have managed to adjust the code slightly to confirm whether or not this is the correct behavior, but I removed the button from the same line.

In player_toxic.cpp

I removed lines 51 & 53.

It looks like this now:

```
static int wanted_level;
			ImGui::SliderInt("WANTED_LVL"_T.data(), &wanted_level, 0, 5);

			components::player_command_button<"wanted">(g_player_service->get_selected(), {(uint64_t)wanted_level}, "Set");

			components::player_command_button<"remweaps">(g_player_service->get_selected(), {});

			components::player_command_button<"tutorial">(g_player_service->get_selected(), {});
```

### Steps To Reproduce

1. Go online
2. Click on a player
3. Navigate to "Set wanted level"
4. Try to apply it.

### Expected Behavior

I'm imaging it makes sense for the button to be lower than the slider due to the visible estate of the UI.

### Screenshots and/or Logs

```
None.
```

### Store

Steam

### Additional context

Player UI tweaks (https://github.com/YimMenu/YimMenu/pull/1489) -> redesigned UI

### Build

- [X] I've confirmed the issue exists on the latest version of YimMenu

### PR DESCRIPTION
**Pull request by Tayipc38**
I have replaced majority of `ImGui::Separator` with `ImGui::SeparatorText`, this way some space is saved and it looks relatively prettier.

Added color change for hovered buttons and changed the default gui colors to a better looking one.

Fixes #1531 
Fixes #1534 
Closes #1528

### CODE REVIEW

File: src/views/players/player/player_toxic.cpp [hunk 1]
```diff
@@ -47,10 +47,11 @@ namespace big
 			components::player_command_button<"fakeban">(g_player_service->get_selected(), {});
 
 			static int wanted_level;
-			ImGui::SliderInt("WANTED_LVL"_T.data(), &wanted_level, 0, 5);
+			components::small_text("WANTED_LVL"_T.data());
```
Comment (Yimura) [inline src/views/players/player/player_toxic.cpp]: `.data()` is unnecessary, components::small_text should have support for `std::string_view`.

File: src/views/world/view_world.cpp [hunk 2]
```diff
@@ -1,34 +1,26 @@
...
-        {
-            view::time_and_weather();
-        }
-
-        ImGui::Separator();
-        
-        components::sub_title("GUI_TAB_WATER"_T);
-        {
-		    components::command_checkbox<"partwater">();
-        }
```
Comment (Yimura) [inline src/views/world/view_world.cpp:21]: Where did this go to?

**Review by Yimura — CHANGES_REQUESTED** [review]

File: src/views/world/view_world.cpp (hunk 2 above)
Comment (Tayipc38) [reply src/views/world/view_world.cpp:21]: Its a feature that does not work, said so in the command itself. Looked like waste of space so I removed it.

Comment (Yimura) [reply src/views/world/view_world.cpp:21]: Makes sense.

**Comment by sch-lda** [comment]
image
This feature (part water) used to work properly until commit d45d0c1.I'm not sure why commit d45d0c1 modified this file.

**Review by Yimura — DISMISSED** [review]

File: src/core/globals.hpp [hunk 3]
```diff
@@ -727,18 +729,18 @@ namespace big
 
 		struct window
 		{
-			ImU32 background_color = 3189656871;
+			ImU32 background_color = 3696311571;
 			ImU32 text_color       = 4294967295;
-			ImU32 button_color     = 4216956014;
-			ImU32 frame_color      = 2939772740;
+			ImU32 button_color     = 2947901213;
+			ImU32 frame_color      = 2946802852;
```
Comment (maybegreat48) [inline src/core/globals.hpp]: The frame color is currently way too bright. I recommend using something darker like `2942518340` or `2940549947` instead

Comment (Tayipc38) [reply src/core/globals.hpp]: Looked great on my end.

**Review by Yimura — APPROVED** [review]

**Pull request merged**